### PR TITLE
Add few guides on advanced configurations

### PR DESCRIPTION
**Pull request by gomathyk**
Add remaining files to advanced section

### CODE REVIEW

File: en/docs/guides/login/session-timeout.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Follow the steps given below to change the time for which the SSO session can be idle before the user gets logged out. 
+
+{! fragments/session-timeout.md !}
+
+!!! Tip "Configure timeout values globally" 
+    The global configuration can be added in the
+    `<IS_HOME>/repository/conf/deployment.toml` file as shown below.
+    ```
+    [session.timeout]
+    idle_session_timeout= "15m"
```
Comment (DMHP) [inline en/docs/guides/login/session-timeout.md:12]: Let's explain the two properties 'idle_session_timeout' and 'remember_me_session_timeout'

File: en/docs/guides/login/use-certificates-with-applications.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Use Certificates with Applications
+
+If you want to configure your application to use a public certificate instead of the default JWKS endpoint, follow the steps given below.
```
Comment (DMHP) [inline en/docs/guides/login/use-certificates-with-applications.md:3]: Let's mention the use cases where we need this feature.

1. OIDC Token Encryption
2. Request Object flow

File: en/docs/guides/login/use-certificates-with-applications.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+3.  Paste the public certificate you obtained in the X509 format above in the **Value** text area. 
+
+    ![oidc-certificate](../../assets/img/guides/oidc-certificate.png)
+
+4.  Click **Update**.
+
+!!! info "What's next?" 
+    -   [Generate and Encrypt ID Tokens](../../../guides/login/oidc-token-encryption)
+
+    -   [Try our sample](../../../quick-starts/oidc-token-encryption-sample)
```
Comment (DMHP) [inline en/docs/guides/login/use-certificates-with-applications.md:20]: Let's link JWKS endpoint docs too

File: en/docs/guides/login/use-certificates-with-applications.md (hunk 2 above)
Comment (sherenewso2) [inline en/docs/guides/login/use-certificates-with-applications.md:3]: Follow the steps given below if you wish to configure...

File: en/docs/guides/login/session-timeout.md (hunk 1 above)
Comment (gomathyk) [reply en/docs/guides/login/session-timeout.md:31]: Modified with a one-line explanation.

File: en/docs/guides/login/use-certificates-with-applications.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/guides/login/use-certificates-with-applications.md:26]: Linked to the said file.

File: en/docs/guides/login/use-certificates-with-applications.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # Use Certificates with Applications
+Follow the steps given below if you wish to configure your application to use a public certificate instead of the default JWKS endpoint. This configuration is required to follow the scenarios mentioned below. 
 
-If you want to configure your application to use a public certificate instead of the default JWKS endpoint, follow the steps given below.
+1.  OIDC Token Encryption
```
Comment (DMHP) [inline en/docs/guides/login/use-certificates-with-applications.md:4]: Shall we link to the relevant guide pages for these features?

**Pull request merged**